.gitignore

```diff
@@ -14,3 +14,5 @@ dist
 coverage
 
 build.tar.gz
+/test/cassettes/Referral*/creates-a-referral-user*
+/test/cassettes/Referral*/add-a-referral-user-credit-card*
```

Comment:
As long as the tests are skipped, I don't see a need to include this. It frankly could frustrate the next person coming along trying to commit these (when we are able to redact info) and they can't find why these disappeared. Thoughts?

Reply:
Sounds good